pkg/kf/apps/kfapp.go

```diff
@@ -45,28 +46,35 @@ func (k *KfApp) GetNamespace() string {
 	return k.Namespace
 }
 
-func (k *KfApp) getOrCreateRunLatest() *serving.RunLatestType {
-	if k.Spec.RunLatest == nil {
-		k.Spec.RunLatest = &serving.RunLatestType{}
+func (k *KfApp) getOrCreateRunLatest() *serving.RevisionTemplateSpec {
```

Comment:
Given that `RunLatest` is no longer a concept in knative, does this method name still make sense?